UI/Selenium/Steps/ConsultationRoomSteps.cs

```diff
@@ -63,8 +93,21 @@ public void ThenJudgeChecksParticipantJoinedTheConsultationRoom()
             var participant = _hearing.Participant.FirstOrDefault(p => !p.Id.ToLower().Contains("judge"));
             var participantName = participant.Name;
             ExtensionMethods.FindElementWithWait(Driver, ConsultationRoomPage.ParticipantTick(participantName.FirstName), _scenarioContext, TimeSpan.FromSeconds(Config.DefaultElementWait));
-            var isTickVisible = ExtensionMethods.IsElementVisible(Driver, ConsultationRoomPage.ParticipantTick(participantName.FirstName), _scenarioContext);
+            var isTickVisible = ExtensionMethods.IsElementVisible(Driver, ConsultationRoomPage.ParticipantTick(participantName.FirstName),_scenarioContext);
+        }
```

Comment:
You have created the var isTickVisible - are you going to do anything with it?

Reply:
Corrected